### PR TITLE
Add power battery% access 

### PR DESCRIPTION
**Pull request by StraToN**
Only tested X11 and Windows by @vnen. Sorry I don't have a mac
JOB IS NOT FINISHED FOR ALL PLATFORMS! NEED TESTERS for iOS, OSX, WinRT and Android (may not compile for these).
Thanks

### CODE REVIEW

File: platform/android/power_android.cpp [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+#include "power_android.h"
+
+/* returns 0 on success or -1 on error (others undefined then)
+ * returns truthy or falsy value in plugged, charged and battery
+ * returns the value in seconds and percent or -1 if not available
+ */
+
+int Android_JNI_GetPowerInfo(int* plugged, int* charged, int* battery, int* seconds, int* percent)
+{
+	/*
```
Comment (bojidar-bg) [inline platform/android/power_android.cpp]: I'm not really sure this should be commented :)

Comment (StraToN) [inline platform/android/power_android.cpp]: I believe this directly comes from SDL @bojidar-bg 
I'll remove it if you think it has to be.

Comment (bojidar-bg) [inline platform/android/power_android.cpp]: Well, it's mostly useless to have a function that has a only a `return true` inside it...

File: AUTHORS [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Godot Engine is a project initiated by Juan "reduz" Linietsky and Ariel "punto" Manzur.
```
Comment (akien-mga) [inline AUTHORS]: I think this should probably be added in another PR, with a first attempt at writing the mentioned exhaustive list of contributors.

File: platform/android/power_android.cpp (hunk 1 above)
Comment (akien-mga) [inline platform/android/power_android.cpp]: BTW, it would be nice to make it very clear in all files which code chunks come from SDL (and maybe even the version) so that we can compare with upstream and do changes as needed in the future.
If complete files come from SDL and were just "ported" to Godot's standard library, you could put this in a comment just below the Godot license header.

Comment (StraToN) [inline platform/android/power_android.cpp]: This function was imported from SDL but I am unable to test it so I commented it out. It's here to be filled in the future.
Also, I'm unable to get this branch updated with current master in order to shoot down conflicts, so I'll close this PR and remake it completely from scratch.
